front/components/assistant_builder/ActionScreen.tsx

```diff
@@ -641,6 +575,72 @@ export default function ActionScreen({
             canSelectTable={dataSources.length !== 0}
           />
         </ActionModeSection>
+
+        <ActionModeSection show={builderState.actionMode === "DUST_APP_RUN"}>
+          {noDustApp ? (
+            <ContentMessage
+              title="You don't have any Dust Application available"
+              variant="warning"
+            >
+              <div className="flex flex-col gap-y-3">
+                {(() => {
+                  switch (owner.role) {
+                    case "admin":
+                    case "builder":
+                      return (
+                        <div>
+                          <strong>
+                            Visit the "Developer Tools" section in the Build
+                            panel to build your first Dust Application.
+                          </strong>
+                        </div>
+                      );
+                    case "user":
+                      return (
+                        <div>
+                          <strong>
+                            Only Admins and Builders can build Dust
+                            Applications.
+                          </strong>
+                        </div>
+                      );
+                    case "none":
+                      return <></>;
+                    default:
+                      assertNever(owner.role);
+                  }
+                })()}
+              </div>
+            </ContentMessage>
+          ) : (
+            <>
+              <div className="text-sm text-element-700">
+                The assistant will execute a{" "}
+                <a
+                  className="font-bold"
+                  href="https://docs.dust.tt"
+                  target="_blank"
+                >
+                  Dust Application
+                </a>{" "}
+                of your design before replying. The output of the app (last
+                block) is injected in context for the model to generate an
+                answer. The inputs of the app will be automatically generated
+                from the context of the conversation based on the descriptions
+                you provided in the application's input block dataset schema.
```

Comment:
Not in figma; should we keep it?

Reply:
Nothing makes it impossible for anyone to even find a place to read about this. We need to come up with a documentation + link it from here.

We'll likely do the same for Query Tables (at least that's the plan as per the project issues).

Not doing it here but def TODO, but removing everything means there's no way to learn about his at all..,